test/agentchat/test_tool_calls.py

```diff
@@ -75,6 +75,62 @@ def test_eval_math_responses():
     print(eval_math_responses(**arguments))
 
 
+@pytest.mark.skipif(skip_openai or not TOOL_ENABLED, reason="openai>=1.1.0 not installed or requested to skip")
+def test_eval_math_responses_api_style_function():
+    config_list = autogen.config_list_from_models(
+        KEY_LOC,
+        model_list=["gpt-4-0613", "gpt-3.5-turbo-0613", "gpt-3.5-turbo-16k"],
```

Comment:
On Azure, a deployment (model) cannot have "." in the name. It should be (most likely because you can give an arbitrary name to a deployment):
```python
model_list=["gpt-4-0613", "gpt-35-turbo-0613", "gpt-35-turbo-16k"],
```